hydra-node/test/Hydra/NodeSpec.hs

```diff
@@ -13,16 +14,43 @@ import Hydra.HeadLogic (
   Event (..),
   HeadState (..),
  )
-import Hydra.Ledger (Party, SigningKey, Tx, deriveParty, sign)
+import Hydra.Ledger (Tx)
 import Hydra.Ledger.Simple (SimpleTx (..), simpleLedger, utxoRef, utxoRefs)
 import Hydra.Logging (Tracer, showLogsOnFailure)
 import Hydra.Network (Host (..), Network (..))
 import Hydra.Network.Message (Message (..))
-import Hydra.Node (EventQueue (..), HydraNode (..), HydraNodeLog, createEventQueue, createHydraHead, isEmpty, stepHydraNode)
+import Hydra.Node (EventQueue (..), HydraNode (..), HydraNodeLog, createEventQueue, createHydraHead, initEnvironment, isEmpty, stepHydraNode)
+import Hydra.Options (Options (..), defaultOptions)
+import Hydra.Party (Party, SigningKey, alias, deriveParty, generateKey, sign)
 import Hydra.Snapshot (Snapshot (..))
+import System.FilePath ((</>))
+import System.IO.Temp (withSystemTempDirectory)
 
 spec :: Spec
 spec = parallel $ do
+  describe "initEnvironment" $ do
+    -- TODO(SN): maybe we should rather create some 'loadParty' from file
+    -- functions and unit test them instead?
```

Comment:
Suggestion: Could be a property test: Generate file names and check those starting with `isLetter` have a proper `Party`?

Reply:
You mean on this level or on top of my TODO of extracting `loadParty`?

Reply:
The latter, yes.